examples/code_browser.py

```diff
@@ -0,0 +1,55 @@
+import sys
+
+from rich.syntax import Syntax
+from rich.traceback import Traceback
+from textual.app import App, ComposeResult
+from textual.layout import Container, Vertical
+from textual.reactive import Reactive
+from textual.widgets import DirectoryTree, Footer, Header, Static
+
+
+class CodeBrowser(App):
+
+    show_tree = Reactive.init(True)
+
+    def watch_show_tree(self, show_tree: bool) -> None:
+        self.set_class(show_tree, "-show-tree")
+
+    def on_load(self) -> None:
+        self.bind("t", "toggle_tree", description="Toggle Tree")
+        self.bind("q", "quit", description="Quit")
+
+    def compose(self) -> ComposeResult:
+        path = "./" if len(sys.argv) < 2 else sys.argv[1]
```

Comment:
If you run this with `textual run examples/code_browser.py`, it fails with `FileNotFoundError: [Errno 2] No such file or directory: 'run'`

Reply:
Works now, but you will need to wrap the args in quotes. i.e. `textual run "code_browser ../"`